bees/ircbee/ircbee.go

```diff
@@ -125,6 +125,42 @@ func (mod *IrcBee) part(channel string) {
 	}
 }
 
+func (mod *IrcBee) statusChange(eventChan chan bees.Event,conn *irc.Conn, line *irc.Line) {
+	//Line.CMD eq Handler Name ex: JOIN
+	message := ""
+	switch line.Cmd {
+	case "JOIN":
+		message = "User joined to channel " + line.Args[0]
+	case "PART":
+		message = "User parted to channel " + line.Args[0]
+	case "QUIT":
+		message = "User quit from channel " + line.Args[0]
+	default:
+		panic("Unknown command " + line.Cmd + " in statusChange")
```

Comment:
I don't think we should panic() here. It's just a message we don't handle / we're not interested in (yet). You can print out an error or even silently ignore it.